code/modules/projectiles/ammunition/ammo_casings.dm

```diff
@@ -243,7 +243,7 @@
 	name = "rubber shot"
 	desc = "A shotgun casing filled with densely-packed rubber balls, used to incapacitate crowds from a distance."
 	icon_state = "bshell"
-	projectile_type = /obj/item/projectile/bullet/pellet/shotgun_rubbershot
+	projectile_type = /obj/item/projectile/bullet/rpellet/shotgun_rubbershot
```

Comment:
Instead of changing the type refactor the shotgun nerf into a prco and override it on shotgun_rubbershot

Reply:
I have no clue how would I do that to be honest, I'm a newbie when it comes to coding.